csharp-advanced-topics/DelegatesInCsharp/ActionFuncTests/OtherTests.cs

```diff
@@ -0,0 +1,60 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Linq;
+
+namespace ActionFunc.Tests
+{
+    [TestClass()]
+    public class OtherTests
+    {
+        [TestMethod()]
+        public void AnonymousFuncTest()
+        {
+            try
+            {
+                int a = 3;
+                int b = 9;
```

Comment:
We use var to define variables unless we must use the type explicitly or it's better to use it to make the examples clearer for the reader. In short, please use vars unless you need to use the type explicitely.